source/Sashimi/Sashimi.csproj.DotSettings

```diff
@@ -0,0 +1,2 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
+	<s:Boolean x:Key="/Default/CodeInspection/NamespaceProvider/NamespaceFoldersToSkip/=actionhandlers/@EntryIndexedValue">True</s:Boolean></wpf:ResourceDictionary>
```

Comment:
Am I correct in thinking we don't want to commit this file?

Reply:
used to mark one of the folders in the project as not a namespace provider